asset/css/page-nav.css

```diff
@@ -35,10 +35,11 @@
 
 /* Responsive site navigation */
 
-/* Hides page navigation on screen widths smaller than 1300px */
-@media screen and (max-width: 1299.98px) {
+/* Hides page navigation on screen widths smaller than 1300px, or when printing */
+@media screen and (max-width: 1299.98px), print {
 
     #page-nav {
+        border: none;
```

Comment:
Good addition, didn't think about the border being visible when page navigation is hidden.